jabkit/src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -164,7 +164,7 @@ protected static void saveDatabaseContext(CliPreferences cliPreferences,
             if (!FileUtil.isBibFile(outputFile)) {
                 System.err.println(Localization.lang("Invalid output file type provided."));
             }
-            System.out.println(Localization.lang("Saving: %0.", outputFile));
+            System.out.println(Localization.lang("Saved %0.", outputFile));
```

Comment:
Nitpick: The output of the state should be AFTER it happend. Just move it just before the "} catch (IOException ex) {"

(it seems the code it not from you, but I first saw it in your PR ^^)